layouts/index.html

```diff
@@ -59,6 +66,10 @@ <h2 class="title--answer">Mostly...</h2>
         <a href="https://gitlab.gnome.org/GNOME/eog">Eye of GNOME</a>,
         <a href="https://github.com/eXeC64/imv">imv</a>
       </li>
+      <li class="list__item--ok">
+        Logout Menu:
```

Comment:
Can we rename to "Power menu"? I think it is better if a bit more generic

Reply:
yeah cool